packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts

```diff
@@ -25,7 +25,7 @@ describe('LWC Definition Linking', () => {
 
   it('Should provide navigation to a selected LWC tag', async () => {
     // select the 'c-view-source' tag
-    testDefinitionNavigation(
+    return testDefinitionNavigation(
```

Comment:
for tests that have async on the funciton we either need to return a promise or use the done function.  I just updated to return the promise here.